cypress/support/pages/articles.pageObject.js

```diff
@@ -0,0 +1,23 @@
+import PageObject from '../PageObject';
+
+class ArticlesPageObject extends PageObject {
+  url = '/#/articles/';
```

Comment:
In the `url` assignment, it's a good practice to not hardcode the URLs in the page objects. Consider using a config file or environment variables to store URLs.